src/components/input/input-checkbox-radio-styles.jsx

```diff
@@ -0,0 +1,137 @@
+import { createStyleSheet } from 'jss-theme-reactor';
+import variables from '../../utilities/variables';
+
+export default createStyleSheet('InputCheckboxRadio', (theme) => {
+  const { palette } = theme;
+
+  const modifierFn = color => ({
```

Comment:
We no longer need BEM approach for styles. Even nesting class selectors might not be needed as extensively anymore.